net-dns/dnsdist/dnsdist-1.9.8-r2.ebuild

```diff
@@ -49,7 +49,10 @@ RDEPEND="acct-group/dnsdist
 DEPEND="${RDEPEND}"
 BDEPEND="virtual/pkgconfig"
 
-PATCHES=( "${FILESDIR}/1.9.8-quiche-0.23.patch" )
+PATCHES=(
+	"${FILESDIR}/1.9.8-quiche-0.23.patch"
+	"${FILESDIR}/1.9.8-fix-compat-with-boost-lockfree-1.87.0.patch"
```

Comment:
I assume you're revbumping here because it affects an installed header?

Reply:
Well, I saw the patch to a .h, and guessed ;)

If it changes the compiled code, then revbump is OK (just perhaps note that in the commit message).

Reply:
OK, I'll amend the commit message.

Reply:
I hope the comment clarifies things.